src/components/Home/TopVoters.tsx

```diff
@@ -27,8 +28,13 @@ function TopVoters() {
       <Table basic="very" unstackable>
         <Table.Header>
           <Table.Row>
-            <Table.HeaderCell>
+            <Table.HeaderCell className="TopVoters__HeaderTitle">
               <Header sub>{t('page.home.community_engagement.top_voters')}</Header>
+              <Helper
+                text={t('page.home.community_engagement.top_voters_tooltip', { vp: TOP_VOTER_VP_THRESHOLD })}
+                position={'top center'}
```

Comment:
Is this size and position ok in mobile?

Reply:
LGTM

![image](https://user-images.githubusercontent.com/2858950/228341208-3c338bfb-8e9a-4d0c-a8c8-fea41525599f.png)